src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DeleteOperationTest.java

```diff
@@ -38,6 +40,7 @@ public class DeleteOperationTest extends AbstractValidatingStargateBridgeTest {
 
   @Inject QueryExecutor queryExecutor;
   @Inject ObjectMapper objectMapper;
+  @Inject DocumentConfig documentConfig;
```

Comment:
I think we don't need this can control in the tests ourself

Reply:
Not sure why can't be used. Anyway removed it as suggested.

Reply:
Because test depends not on it, you can add retry values yourself in the test.. What s the point of `@Injecting` this?